### PR TITLE
submit lesson 2 assignment 1 (calculator)

### PR DESCRIPTION
**Pull request by ZomboFunk**

### CODE REVIEW

**Comment by ZomboFunk** [comment]
I'm not 100% sure I understood what we were supposed to do. I couldn't get over the whole, "make a calculator" concept. I hope this counts.

File: Lesson 02/countdown.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+let i = 10
```
Comment (jimmynono) [inline Lesson 02/countdown.txt]: You're missing a semi colon here.

File: Lesson 02/countdown.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+while (i >= 0){
```
Comment (jimmynono) [inline Lesson 02/countdown.txt]: You'll want an empty space before the curly brace;

File: Lesson 02/countdown.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  console.log(i);
+  i--
```
Comment (jimmynono) [inline Lesson 02/countdown.txt]: You're missing a semi colon here

**Comment by ZomboFunk** [comment]
Thanks for the feedback. I didn't realize there were any comments until just now. I definitely need to work on the JS 'grammar'. =[

File: Lesson 02/calc-2.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+let a = 1;
+let b = 2;
+let c = 3;
+let d = 4;
+console.log(a+b)
```
Comment (jimmynono) [inline Lesson 02/calc-2.js]: You'l want to put an empty line after this block of variable declarations

File: Lesson 02/calc-2.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+console.log(b*c)
```
Comment (jimmynono) [inline Lesson 02/calc-2.js]: You'll want to put a space before and after operators...

console.log(b * c);

**Comment by jimmynono** [comment]
> Thanks for the feedback. I didn't realize there were any comments until just now. I definitely need to work on the JS 'grammar'. =[

That's the goal of this class. =]

**Comment by ZomboFunk** [comment]
Great news! I can still submit pull requests to this repository. I still haven't quite figured out what is wrong with the other one but I'm glad it doesn't impact my ability to submit these.

File: Lesson 04/square-it.html [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+<html>
+<body>
+
+<h2>SQUARE IT!</h2>
+
+<button onclick="myFunction()">Square it</button>
+
+<script>
+function myFunction() {
+  var txt;
```
Comment (jimmynono) [inline Lesson 04/square-it.html]: Don't use var. Use let or const for variables

File: Lesson 04/square-it.html [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  var number = prompt("Please enter a number", "0");
+  if (number == null || number == ""){
```
Comment (jimmynono) [inline Lesson 04/square-it.html]: You will pretty much always want to use === for our assignments.

File: Lesson 04/square-it.html [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+  txt = "user didn't input a number";
+  } else {
+    var answer = alert(number*number);
```
Comment (jimmynono) [inline Lesson 04/square-it.html]: Make sure and put spaces around your operators
```
var answer = alert(number * number);
```

Comment (ZomboFunk) [reply Lesson 04/square-it.html]: ohhh... that's probably why it didn't work the first couple of times. I was originally trying it a different way but it wasn't working. Thanks!

**Pull request closed without merging**